pkg/hub/client/public/fields.go

```diff
@@ -19,7 +19,7 @@ var GenericMetadataFields = `
       iconURL
       `
 
-// AttributeFields for quering the Attributes fields and GenericMetadata.
+// AttributeFields for querying the Attributes fields and GenericMetadata.
 var AttributeFields = fmt.Sprintf(`
```

Comment:
BTW I think we can unexport the `AttributeFields` and `GenericMetadataFields`, as they are not used anywhere else than in this package (and IMHO they shouldn't).